public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
Blocking bots isn't inside the scope of the Jira story. Ordinarily, we wouldn't want out-of-scope changes to be included in a PR (unless they're minor). This is quite minor, so no need to change anything.

This is a tip for the future.